x-pack/solutions/search/plugins/enterprise_search/server/deprecations/index.ts

```diff
@@ -0,0 +1,366 @@
+      ];
+    },
+  };
+};
+
+/**
+ * If Enterprise Search Node is configured, it's marked as a critical deprecation
+ * Warns that removing the node will disable crawlers/connectors
+ */
+export function getEnterpriseSearchNodeDeprecation(
+  config: ConfigType,
+  isCloud: boolean
+): DeprecationsDetails[] {
+  if (config.host) {
+    const steps = [];
+    if (isCloud) {
+      steps.push(
+        ...[
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.gotocloud', {
+            defaultMessage: 'Go to cloud.elastic.co',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clickedit', {
+            defaultMessage: "Click the 'Edit' tab",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.scrolldown', {
+            defaultMessage: "Scroll down to the 'Enterprise Search' section",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clickx', {
+            defaultMessage: "Click the red 'X' by 'Enterprise Search instances'",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.nocapacity', {
+            defaultMessage: 'You should no longer see any Enterprise Search capacity',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clicksave', {
+            defaultMessage: "Click 'Save' and confirm",
+          }),
+        ]
+      );
+    } else {
+      steps.push(
+        ...[
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.stopent', {
+            defaultMessage: 'Stop all your Enterprise Search nodes',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.stopkibana', {
+            defaultMessage: 'Stop all your Kibana nodes',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.removeconfig', {
+            defaultMessage: "Edit 'kibana.yml' to remove 'enterpriseSearch.host'",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.restart', {
+            defaultMessage: 'Restart Kibana',
+          }),
+        ]
+      );
+    }
+    return [
+      {
+        level: 'critical',
+        deprecationType: 'feature',
+        title: i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.title', {
+          defaultMessage: 'Enterprise Search host(s) must be removed',
+        }),
+        message: i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.message', {
+          defaultMessage:
+            'Enterprise Search is not supported in versions >= 9.x. ' +
+            'You must remove Enterprise Search node(s) from your deployment to proceed with the upgrade. ' +
+            'Note that once Enterprise Search is stopped, products such as App Search, Workplace Search, and Elastic Crawler ' +
```

Comment:
Nit, I'd like to see the following 3 pieces of information in a bulleted list for readability, but I totally understand that might not have the time for 💅 the formatting here :)

So ideally this section would look something like this:

```markdown
Enterprise Search is not supported in versions >= 9.x. 

Please note the following:

- You must remove any Enterprise Search nodes from your deployment to proceed with the upgrade.
- Once Enterprise Search is stopped, products such as App Search, Workplace Search, and Elastic Crawler will cease to operate.
- Native Connectors will also stop running syncs until the 9.x upgrade completes.
```

Reply:
I'll give it a shot, but will it be ok if it's ascii bullets like this, and not actual HTML bullets?

I don't have any formatting controls here, AFAIK. I'm just returning a JSON object, where the string value gets dumped to the page.

Reply:
As I mentioned, I'll leave it up to you considering time/complexity tradeoff :)

Reply:
Ok, after learning that markdown was an option, I've taken another pass adding whitespace, and I've updated the screenshots in the description of this PR. Let me know what you think!

Reply:
The new .md version looks great!